spec/controllers/idv/in_person/address_controller_spec.rb

```diff
@@ -17,15 +17,22 @@
       and_return(true)
     allow(subject).to receive(:current_user).
       and_return(user)
-    allow(subject).to receive(:pii_from_user).and_return(pii_from_user)
-    allow(subject).to receive(:flow_session).and_return(flow_session)
     stub_sign_in(user)
+    subject.user_session['idv/in_person'] = flow_session
```

Comment:
Since we're removing references to `flow_session` below, can we get rid of the `let` as well and just initialize this:

```suggestion
    subject.user_session['idv/in_person'] =  { pii_from_user: pii_from_user }
```